protos/dcs/coalition/v0/coalition.proto

```diff
@@ -121,6 +124,39 @@ message AddGroupResponse {
   dcs.common.v0.Group group = 1;
 }
 
+message AddStaticObjectRequest {
+  // the name of the static; must be unique or would destroy previous object
+  string name = 1;
+  // country the unit belongs to
+  dcs.common.v0.Country country = 2;
+  // type of the static object (e.g. "Farm A", "AS32-31A")
+  string type = 3;
+  // string name of the livery for the aircraft
+  string livery = 4;
+  // boolean for whether or not the object will appear as a wreck
+  bool dead = 5;
+  // number value for the "score" of the object when it is killed
+  optional uint32 score_rate = 6;
+
+  double heading = 7;
+  dcs.common.v0.Position position = 8;
+  Offset offset = 9;
+
+  message Offset {
+    // the name of the unit to offset from
+    string unit = 1;
+    double x = 2;
+    double y = 3;
+  }
+
+  // cargo mass in kilograms
+  uint32 cargo_mass = 10;
```

Comment:
There are a lot variations on the behavior, depending on the scenario. Examples are:

* Heading/angle is orientated based off the linked unit's heading
* Cargo is only allowed on non-linked statics

An alternative I was debating was the following pseudo:
```
oneof position {
   StaticPositioning static = YYY;
   LinkedPositioning offset = YYYY;
}

message StaticPositioning {
  double heading = 7;
  dcs.common.v0.Position position = 8;
  uint32 cargo_mass = 10;
}

message Offset {
  // the name of the unit to offset from
  string unit = 1;
  double x = 2;
  double y = 3;
}
```
Alternatively, two discrete API's for `AddStatic` and `AddLinkedStatic`

Reply:
From what I understand by looking at the hoggitworld wiki it looks like the offset / linking stuff is to put statics on ships... carriers, really with planes, cranes etc.

Given this fact and my impression that this "ship linking" offset stuff will be used a _lot_ less than non-linking stuff I think your suggestion of two separate APIs is a good one.